### PR TITLE
Document several worker status endpoints using OpenAPI

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

**Review by bmwiedemann — APPROVED** [review]

File: src/api/public/apidocs/paths/build__workerstatus.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  deprecated: true
+  summary: Lists status of workers, jobs, backend services and general statistics.
+  description: |
+    Lists status of workers, running jobs, waiting jobs, status of the backend services and general statistics.
+
+    Other ways to obtain the same information as with this endpoint are:
+      * `GET /worker/_status`
+      * `GET /status/workerstatus`
+      * `GET /build/_workerstatus`
+      * `GET /worker/status`
```
Comment (eduardoj) [inline src/api/public/apidocs/paths/build__workerstatus.yaml:11]:
```suggestion
    This endpoint is exactly the same as `GET /worker/_status`, please use that one.
```
For a deprecated endpoint, we should recommend using the endpoint which is not deprecated. No need to also list the already deprecated alternatives.

The same applies for the rest of the deprecated endpoints.

File: src/api/public/apidocs/paths/worker__status.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+get:
+  summary: Lists status of workers, jobs, backend services and general statistics.
+  description: |
+    Lists status of workers, running jobs, waiting jobs, status of the backend services and general statistics.
+
+    Other ways to obtain the same information as with this endpoint are:
+      * `GET /worker/_status`
+      * `GET /status/workerstatus`
+      * `GET /build/_workerstatus`
+      * `GET /worker/status`
```
Comment (eduardoj) [inline src/api/public/apidocs/paths/worker__status.yaml:10]:
```suggestion
```
There is no need to list the alternatives if they all deprecated.

**Comment by eduardoj** [comment]
I rebased with master and added a commit to address the feedback of the review, to be merged with the original commit.

File: src/api/public/apidocs/paths/worker__status.yaml (hunk 2 above)
Comment (eduardoj) [reply src/api/public/apidocs/paths/worker__status.yaml]: Addressed.l

File: src/api/public/apidocs/paths/build__workerstatus.yaml (hunk 1 above)
Comment (eduardoj) [reply src/api/public/apidocs/paths/build__workerstatus.yaml]: Addressed.

**Comment by krauselukas** [comment]
@eduardoj just squash the commits and I will approve :)

File: src/api/public/apidocs/paths/status_workerstatus.yaml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                    starttime: '1617148925'
+                    queue:
+                      high: '0'
+                      med: '1'
+                      low: '4'
+                      next: '22'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+  tags:
+    - Status
```
Comment (krauselukas) [inline src/api/public/apidocs/paths/status_workerstatus.yaml:109]:
```suggestion
    -  Status Project
```

**Review by krauselukas — APPROVED** [review]

**Review by krauselukas — APPROVED** [review]

**Pull request merged**